docs/users.rst

```diff
@@ -625,3 +625,37 @@ same effect, specify such images with another stage, for example::
     COPY --from=source1 src/ dest/
 
 
+Operator manifests
+------------------
+
+An operator_ is an image with additional metadata. This image should contain a
```

Comment:
`An operator_ is an image with additional metadata.` Please drop this sentence or use something from https://coreos.com/operators/

Reply:
Maybe `OSBS is able to extract operator manifests from an operator image.`   [operator image] can be linked to https://coreos.com/operators/

Reply:
Done